vmdb/app/models/miq_schedule.rb

```diff
@@ -334,8 +334,8 @@ def depot_hash
   end
 
   def next_interval_time
-    unless self.valid? || self.errors.on(:run_at).empty?
-      $log.warn("Miq(Schedule.next_interval_time) Invalid schedule [#{self.id}] [#{self.name}]: #{self.errors.on(:run_at).to_miq_a.join(", ")}")
+    unless self.valid? || self.errors.get(:run_at).nil? || self.errors.get(:run_at).empty?
```

Comment:
It's the `unless`-`||` combination.  I find the moment you don't do a plain `unless`, it's better to just do `if !`...even `unless`-`else` is confusing.